lib/matplotlib/tests/test_axes.py

```diff
@@ -22,6 +22,10 @@
 import warnings
 from matplotlib.cbook import IgnoredKeywordWarning
 
+# Note: Some test cases are run twice: once normally and once with labeled data
```

Comment:
it's "only" 15 cases... I didn't double every check, only the most simple one... I've not yet seen how one could use the (=all?)  plot call as a context manager?!